src/components/common/inputs/file-input.jsx

```diff
@@ -67,12 +68,18 @@ export default class FileInput extends React.Component {
           </div>
         )}
 
-        <Button
-          caption={this.props.fileName ? 'Choose Another File' : 'Choose File'}
-          isAllScreens={!this.props.fileName}
-          isFitContent={true}
-          onClick={this.handleButtonClick}
-        />
+        <div className='button-container'>
+          <Button
+            caption={this.props.fileName ? 'Choose Another File' : 'Choose File'}
+            isAllScreens={!this.props.fileName}
+            isFitContent={true}
+            onClick={this.handleButtonClick}
+          />
+
+          {this.props.isLoading && (
+            <Loader className='loader' />
+          )}
```

Comment:
Showing loader in case Google API call takes some time:
![Screen Shot 2022-01-30 at 9 48 54 PM](https://user-images.githubusercontent.com/15594382/151745179-8643f0b8-d07a-43c8-bd91-db7803156ac5.png)